content/_help/trouble-signing-in/face-or-touch-unlock._es.md

```diff
@@ -0,0 +1,62 @@
+    * Su dispositivo debe tener activado el desbloqueo de pantalla.
+
+* Navegadores
+    * Chrome 109 o más reciente
+    * Safari 16 o más reciente
+    * Edge 109 o más reciente
+
+**Importante**: Deberá activar la sincronización con la nube en su dispositivo si desea utilizar el desbloqueo facial o táctil para acceder a Login.gov a través de distintos dispositivos. Es posible que su dispositivo le muestre otras funciones de autenticación, como acceder escaneando un código QR.
+
+Le recomendamos vivamente que añada un segundo método de autenticación por si sustituye o pierde su dispositivo. Si pierde el acceso a su único método de autenticación, deberá eliminar su cuenta y crear una nueva.
+
+### Recibo un mensaje sobre un código QR, clave de seguridad o contraseña. No sé lo que es o no recuerdo haberlo configurado. {#qr-code-info}
+Dependiendo de la combinación de su dispositivo y navegador, es posible que reciba un mensaje sobre un código QR, una clave de seguridad o una contraseña:
+
+* Si recibe un mensaje para escanear un código QR, utilice la cámara de su celular para escanear el código QR que aparece en la pantalla. Le aconsejamos que escanee el código QR con el dispositivo que utilizó originalmente para configurar el desbloqueo facial o táctil.
+* Si recibe un mensaje sobre una clave de seguridad, pero no tiene acceso a una, elija una opción que utilice un dispositivo diferente.
+* Si ve mensajes que mencionan claves de seguridad, se refieren al desbloqueo facial o táctil. Cuando configure el desbloqueo facial o táctil, creará una clave de acceso en su dispositivo.
+
+Si sigue teniendo problemas, es posible que tenga que [eliminar su cuenta](/es/help/manage-your-account/delete-your-account/). [Póngase en contacto con nuestro servicio de asistencia](/es/contact/) si necesita ayuda.
+
+### He perdido mi teléfono o tengo uno nuevo, y ahora no puedo usar el desbloqueo facial o táctil para acceder.
+
+El desbloqueo facial o táctil sólo funcionará en el dispositivo original con el que lo configuró, a menos que configure el desbloqueo facial o táctil en un dispositivo y un navegador más recientes que admitan [claves de acceso](https://fidoalliance.org/passkeys/), y que tengan activado el bluetooth o la sincronización en la nube entre sus cuentas.
+
+Si el desbloqueo facial o táctil está vinculado a su dispositivo y ya no tiene acceso a él, es posible que tenga que [eliminar su cuenta](/es/help/manage-your-account/delete-your-account/) y crear una nueva. [Póngase en contacto con nosotros](/es/contact/) si necesita ayuda.
+
+Para evitar tener que eliminar su cuenta en el futuro, le recomendamos encarecidamente que configure más de un método de autenticación. De ese modo, dispondrá de un método de autenticación de reserva para acceder a su cuenta en caso que pierda uno.
+
+### Cómo configuro el desbloqueo facial o táctil 
+
+Vaya a [secure.login.gov](https://secure.login.gov/es) y haga clic en "Crear una cuenta".
+
+{: .number-list}
```

Comment:
The numbered list appears differently for Spanish than the other languages.

Spanish has a new line above `{: .number-list}` whereas English and Spanish are missing this. Spanish appears to have the correct formatting.